src/components/Item.jsx

```diff
@@ -51,13 +54,43 @@ const Item = ({
     return estimatedInterval;
   };
 
+  function handleClick(e) {
+    e.preventDefault();
+    const confirmation = window.confirm(
+      `Are you sure you want to delete ${name}?`,
+    );
+
+    if (confirmation) {
+      db.collection('items')
+        .doc(id)
+        .delete()
+        .then(() => {
+          setActionMessage(`${name} successfully deleted!`);
+          setTimeout(() => {
+            setActionMessage(null);
+          }, 10000);
+        })
+        .catch((error) => {
+          console.error('Error removing document: ', error);
```

Comment:
Can we show an error message to the user if the delete fails? In this case, we're logging the error to the console, but not letting the user know something went wrong. They will only see that it looked like it went through, but the item remains.